ironfish/src/workerPool/tasks/decryptNotes.ts

```diff
@@ -19,6 +20,7 @@ export interface DecryptNoteOptions {
 }
 
 export interface DecryptedNote {
+  accountId: string
```

Comment:
Seems like we can get away with not passing the `accountId` through here as its not used in the worker pool job